parser/test/parser_test.c

```diff
@@ -1,21 +1,19 @@
 #include <stdio.h>
 
-#include "parser_wrapper.h"
-#include "parse_node.h"
-#include "utility/stringify.h"
-
-int main(void) {
-    char *test = "SELECT c FROM test;";
-    parse_node *node = parse(test);
-
-    char *quickstep_output = quickstep_stringify(node);
-    printf("%s\n", quickstep_output);
-    free(quickstep_output);
+#include "parser/parser_wrapper.h"
+#include "parser/parse_node.h"
+#include "parser/utility/stringify.h"
 
+void test_parse_tree(char *query) {
```

Comment:
Could you modify the test to verify the output of the parser against a known output, i.e. compare the json_output with a string containing the result you know to be correct.

Reply:
Done!